README.md

```diff
@@ -517,16 +517,20 @@ Now we'll sign the RPM:
 
 ```
 rpm --resign <rpm>.rpm  # --addsign would allow us to apply multiple signatures to the RPM
-rpm -qi<file.rpm>  # should now show that the file is signed
+rpm -qi <file.rpm>  # should now show that the file is signed
 rpm -Kv  # should contain NOKEY errors in the lines containing Signature
 # This is because the the (public) key of the RPM signing key is not present,
 # and must be added to the RPM client config to verify the signature:
-sudo rpm --import <publicKey>.asc
+sudo rpm --import <publicKey>.asc (keys are in the lfs repos)
 rpm -Kv  # Signature lines will now contain OK instead of NOKEY
+rpm qi <file.rpm>  # Copy the KEY ID so you can paste it into the next command
+gpg -k <KEY ID>  # Verify the RPM was signed with the correct key
 ```
 
 You can then proceed with distributing the package, via the "test" or "prod" repo,
 as appropriate.
+   
+:warning: Remember to update your `~/.rpmmacros` file with the correct GPG key ID each time you switch between signing an RPM with the test key (required for `yum-test`) and the prod key (required for `yum`).
```

Comment:
This speaks to signing process so no need for detail, but it might be worth having test- and prod-only signing environments with the correct `.rpmmacros` to avoid the potential for user error here.

Reply:
I updated the comment to be more generic.